Documents/API.md

```diff
@@ -0,0 +1,139 @@
+# HMDA Platform API
+
+## Public HTTP Endpoints
+
+* `/`
+    * `GET` - Root endpoint, with information about the HMDA Platform service. Used for health checks
+    
+    Example response, with HTTP code 200:
+    
+    ```json
+    {
+      "status": "OK",
+      "service": "hmda-api",
+      "time": "2016-06-17T13:54:10.725Z",
+      "host": "localhost"
+    }
+    ```
+    
+    
+
+* `/institutions`
+    * `GET` - List of Financial Institutions
+    
+    Example response, with HTTP code 200: 
+    
+    ```json
+    {
+      "institutions": [
+        {
+          "id": "12345",
+          "name": "First Bank",
+          "status": "active"
+        },
+        {
+          "id": "123456",
+          "name": "Second Bank",
+          "status": "inactive"
+        }
+      ]
+    }
+    ```
+
+* `/institutions/<institution>`
+    * `GET` - Details for Financial Institution
```

Comment:
Nope, no error response yet discussed. Definitely makes sense to do this soon as the front-end can start working on displaying errors where appropriate. Created #433